specification/policyinsights/resource-manager/Microsoft.PolicyInsights/preview/2018-07-01-preview/policyTrackedResources.json

```diff
@@ -0,0 +1,526 @@
+          },
+          {
+            "$ref": "#/parameters/filterParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Query results.",
+            "schema": {
+              "$ref": "#/definitions/PolicyTrackedResourcesQueryResults"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/QueryFailure"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "Query at management group scope": {
+            "$ref": "./examples/PolicyTrackedResources_QueryManagementGroupScope.json"
+          },
+          "Query at management group scope using query paramters": {
+            "$ref": "./examples/PolicyTrackedResources_QueryManagementGroupScopeWithFilterAndTop.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.PolicyInsights/policyTrackedResources/{policyTrackedResourcesResource}/queryResults": {
+      "post": {
+        "operationId": "PolicyTrackedResources_ListQueryResultsForSubscription",
+        "description": "Queries policy tracked resources under the subscription.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/policyTrackedResourcesResourceParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/topParameter"
+          },
+          {
+            "$ref": "#/parameters/filterParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Query results.",
+            "schema": {
+              "$ref": "#/definitions/PolicyTrackedResourcesQueryResults"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/QueryFailure"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "Query at subscription scope": {
+            "$ref": "./examples/PolicyTrackedResources_QuerySubscriptionScope.json"
+          },
+          "Query at subscription scope using query paramters": {
+            "$ref": "./examples/PolicyTrackedResources_QuerySubscriptionScopeWithFilterAndTop.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.PolicyInsights/policyTrackedResources/{policyTrackedResourcesResource}/queryResults": {
+      "post": {
+        "operationId": "PolicyTrackedResources_ListQueryResultsForResourceGroup",

```

Comment:
If you plan to release a new API version, the SDK versions will be bumped to major version anyway, I will leave this to your discretion.
If the operation is not pageable, you can simply remove the `x-ms-pageable` extension, if it is you can request a suppression for the violation (I guess it would be valid in this case)

Reply:
I prefer to keep this convention (it's the same in the other specs for our RP)